analyze_shakespeare.py

```diff
@@ -0,0 +1,134 @@
+import pickle
+import string
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+
+
+common_words = ['i', 'what', 'the', 'at', 'why', 'o', 'that',
+                'and', 'to', 'is', 'a', 'of', 'in', 'my', 'with', 'not',
+                'but', 'thou', 'this', 'for', 'it', 'be', 'as', 'thy', 'so',
+                'thee', 'have', 'shall', 'me', 'you', 'by', 'from', 'if',
+                'she', 'him', 'he', 'hers', 'his', 'will', 'your', 'lord']
+
+romeo_characters = ['Rom', 'Jul', 'Nurse', 'Tyb', 'Ben', 'Par', 'Friar']
+ham_characters = ['Ham', 'Oph', 'Hor', 'Ros', 'Guil', 'Pol', 'Laer']
+
+
+def load_romeo():
+    """
+    Loads text from pickled text file and cuts off the stuff
+    at the beginning that's not important
+    """
+    input_file = open('romeo.txt', 'rb')
+    loaded_text = pickle.load(input_file)
+    loaded_text_truncated = loaded_text[loaded_text.find('SCENE'):]
+    return loaded_text_truncated
+
+
+def load_hamlet():
+    """
+    Loads text from pickled text file and cuts off the stuff
+    at the beginning that's not important
+    """
+    input_file = open('hamlet.txt', 'rb')
+    loaded_text = pickle.load(input_file)
+    loaded_text_truncated = loaded_text[loaded_text.find('SCENE'):]
+    return loaded_text_truncated
+
+
+def edit_text(words):
+    """
+    Takes text, gets rid of punctuation, makes lowercase,
+    and gets rid of whitespace character \r. Returns list
+    of all words
+    >>> edit_text('HI THERE!!! You are so $WEET?')
+    ['hi', 'there', 'you', 'are', 'so', 'weet']
+    """
+    for symbol in string.punctuation:  # removes punctuation
+        words = words.replace(symbol, '')
+    edited_words = words.replace('\r', '')  # removes whitespace character
+
+    edited_words = edited_words.lower()  # changes all words to lowercase
+    final_edit = edited_words.split()  # splits words into list
+    return final_edit
+
```

Comment:
Nice modularization of your code into smaller functions. It's clear what each function does and you are not asking a single function to do too much thing.